### ISSUE TITLE
About the MoNuSeg dataset.

### ISSUE THREAD

**Issue by windygooo** [xq141839/SPPNet#6]
I've seen papers that say the MoNuSeg training set has 30 images and the test set contains 14 images. However, I found the training data downloaded from the challenge website has 37 images? May I ask which case do the experiments in this paper fit?
